ceph_iscsi_config/gateway.py

```diff
@@ -402,13 +402,20 @@ def map_luns(self, config):
 
         lio_root = root.RTSRoot()
 
+        # a dict, key with tpg and storage object name
+        tpg_stg_object_dict = {}
+        for tpg in self.tpg_list:
+
+            for l in tpg.luns:
+                tpg_stg_object_dict[str(tpg.tag) + '-' + l.storage_object.name] = l.storage_object.name
+
         # process each storage object added to the gateway, and map to the tpg
         for stg_object in lio_root.storage_objects:
 
             for tpg in self.tpg_list:
                 self.logger.debug("processing tpg{}".format(tpg.tag))
 
-                if not self.lun_mapped(tpg, stg_object):
```

Comment:
With this patch what is the creation time now?

What part of lun_mapped was the issue? Was it constantly scanning the lun dir in configfs or something else?

Reply:
before:
![image](https://user-images.githubusercontent.com/3916975/51576599-e20d8800-1ef1-11e9-93b1-8f65049a1a4e.png)
after:
![image](https://user-images.githubusercontent.com/3916975/51576622-f81b4880-1ef1-11e9-8b14-700689a75796.png)